src/Features/CSharp/Portable/Completion/SuggestionMode/CSharpSuggestionModeCompletionProvider.cs

```diff
@@ -91,6 +93,20 @@ private bool IsAnonymousObjectCreation(SyntaxToken token)
             return false;
         }
 
+        private bool IsPossibleTupleExpression(SyntaxToken token)
```

Comment:
note that this is not any random stuff in parenthesis, it is specifically    (expr)  as in    var t = (expr);

I think it constrains the occurrence of this scenario. It is also the most common scenario that I see so far - once you type    var t = ( ,   it gets completed into  var t = ( | )   , which is convenient, but results in all tuples to start as parenthesized expressions.